web/app/Helpers/helpers.php

```diff
@@ -12,3 +12,262 @@ function pg_is_error(Throwable $e, string $code) : bool
+        [ 'b' => 'f4ff81', 't' => '000000', ],
+        [ 'b' => 'eeff41', 't' => '000000', ],
+        [ 'b' => 'c6ff00', 't' => '000000', ],
+        [ 'b' => 'aeea00', 't' => '000000', ],
+        [ 'b' => 'ffeb3b', 't' => '000000', ],
+        [ 'b' => 'fffde7', 't' => '000000', ],
+        [ 'b' => 'fff9c4', 't' => '000000', ],
+        [ 'b' => 'fff59d', 't' => '000000', ],
+        [ 'b' => 'fff176', 't' => '000000', ],
+        [ 'b' => 'ffee58', 't' => '000000', ],
+        [ 'b' => 'ffff8d', 't' => '000000', ],
+        [ 'b' => 'ffff00', 't' => '000000', ],
+        [ 'b' => 'ffea00', 't' => '000000', ],
+        [ 'b' => 'ffd600', 't' => '000000', ],
+        [ 'b' => 'ffc107', 't' => '000000', ],
+        [ 'b' => 'fff8e1', 't' => '000000', ],
+        [ 'b' => 'ffecb3', 't' => '000000', ],
+        [ 'b' => 'ffe082', 't' => '000000', ],
+        [ 'b' => 'ffd54f', 't' => '000000', ],
+        [ 'b' => 'ffca28', 't' => '000000', ],
+        [ 'b' => 'ffe57f', 't' => '000000', ],
+        [ 'b' => 'ffd740', 't' => '000000', ],
+        [ 'b' => 'ffc400', 't' => '000000', ],
+        [ 'b' => 'ffab00', 't' => '000000', ],
+        [ 'b' => 'ff9800', 't' => '000000', ],
+        [ 'b' => 'fff3e0', 't' => '000000', ],
+        [ 'b' => 'ffe0b2', 't' => '000000', ],
+        [ 'b' => 'ffcc80', 't' => '000000', ],
+        [ 'b' => 'ffb74d', 't' => '000000', ],
+        [ 'b' => 'ffa726', 't' => '000000', ],
+        [ 'b' => 'ffd180', 't' => '000000', ],
+        [ 'b' => 'ffab40', 't' => '000000', ],
+        [ 'b' => 'ff9100', 't' => '000000', ],
+        [ 'b' => 'ff6d00', 't' => '000000', ],
+        [ 'b' => 'ff5722', 't' => '000000', ],
+        [ 'b' => 'fbe9e7', 't' => '000000', ],
+        [ 'b' => 'ffccbc', 't' => '000000', ],
+        [ 'b' => 'ffab91', 't' => '000000', ],
+        [ 'b' => 'ff8a65', 't' => '000000', ],
+        [ 'b' => 'ff7043', 't' => '000000', ],
+        [ 'b' => 'ff9e80', 't' => '000000', ],
+        [ 'b' => 'ff6e40', 't' => '000000', ],
+        [ 'b' => 'ff3d00', 't' => '000000', ],
+        [ 'b' => 'dd2c00', 't' => '000000', ],
+        [ 'b' => '795548', 't' => '000000', ],
+        [ 'b' => 'efebe9', 't' => '000000', ],
+        [ 'b' => 'd7ccc8', 't' => '000000', ],
+        [ 'b' => 'bcaaa4', 't' => '000000', ],
+        [ 'b' => 'a1887f', 't' => '000000', ],
+        [ 'b' => '8d6e63', 't' => '000000', ],
+        [ 'b' => '607d8b', 't' => '000000', ],
+        [ 'b' => 'eceff1', 't' => '000000', ],
+        [ 'b' => 'cfd8dc', 't' => '000000', ],
+        [ 'b' => 'b0bec5', 't' => '000000', ],
+        [ 'b' => '90a4ae', 't' => '000000', ],
+        [ 'b' => '78909c', 't' => '000000', ],
+        [ 'b' => '9e9e9e', 't' => '000000', ],
+        [ 'b' => 'fafafa', 't' => '000000', ],
+        [ 'b' => 'f5f5f5', 't' => '000000', ],
+        [ 'b' => 'eeeeee', 't' => '000000', ],
+        [ 'b' => 'e0e0e0', 't' => '000000', ],
+        [ 'b' => 'bdbdbd', 't' => '000000', ],
+    ];
```

Comment:
Curious, how did you come up with these?

Reply:
https://github.com/LasseRafn/ui-avatars/blob/master/Utils/Input.php#L71